cmd/operator/main.go

```diff
@@ -69,7 +69,8 @@ func main() {
 		caCert      = flag.String("ca-cert-base64", "", "The base64-encoded certificate authority.")
 		webhookAddr = flag.String("webhook-addr", ":10250",
 			"Address to listen to for incoming kube admission webhook connections.")
-		metricsAddr = flag.String("metrics-addr", ":18080", "Address to emit metrics on.")
+		healthProbeAddr = flag.String("health-probe-addr", ":8080", "Address to outputs health status")
```

Comment:
What's wrong with using metric port for this?

Reply:
Not possible -- we don't serve `metrics-addr` using controller-runtime and there is no way to expose a healthz/readyz handler through controller-runtime. If you feel strongly about this, I propose adding port consolidation to our backlog.

Do you have any suggestions for a name other than "heath-probe-address"? They're called "health endpoints" generally (https://kubernetes.io/docs/reference/using-api/health-checks/) and this is the same language used in controller-runtime.

Reply:
What about ignoring that controller runtime flow and implementing our own handlers (using started checkers?) in metric-addr port? (maybe stupid suggestion, I didn't look deeper what else that "AddReady/Health*" methods do..

If it does not make sense, then "probes-addr" might do the work, WDYT?